tests/create_sequence_data.py

```diff
@@ -15,6 +17,7 @@ def create_sequence_data(
     use_mem_mapped=False,
     t_end=10.0,
     sampling_rate=10.0,
+    contain_nans=False,
```

Comment:
future note - add a test case when start time is non-zero (might cause numerics issues)

Reply:
+ add irregular option for timestamps